apps/src/eval/eval.js

```diff
@@ -397,12 +408,24 @@ Eval.execute = function() {
     result: Eval.result,
     testResult: Eval.testResults,
     program: encodeURIComponent(textBlocks),
-    onComplete: onReportComplete
+    onComplete: onReportComplete,
+    image: Eval.encodedFeedbackImage
   };
 
-  studioApp.playAudio(Eval.result ? 'win' : 'failure');
+  if (typeof document.getElementById('svgEval').toDataURL === 'undefined') { // don't try it if function is not defined
```

Comment:
I suspect this is test environment only, though I haven't taken the time to investigate (I just copied this code over from Studio). I might update the comment to say I suspect it's test only.